msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/AbstractMsalUiTest.java

```diff
@@ -90,8 +91,12 @@ public void setup() {
         mScopes = getScopes();
         mBrowser = getBrowser();
 
-        // clear all cookies in the browser
-        mBrowser.clear();
+        // clear all cookies in the browser and in the WebView
+        try {
+            mBrowser.clear();
+            CommonUtils.clearCookiesSync(mActivity);
```

Comment:
Why do we need this? The tests that we have today are run with ANDROID TEST ORCHESTRATOR so the entire storage is wiped between each test. Is that not working as expected?